Tasks/AzureRmWebAppDeployment/azurermwebappdeployment.ts

```diff
@@ -220,4 +234,25 @@ async function updateScmType(SPN, webAppName: string, resourceGroupName: string,
     }
 }
 
+function addWebConfigFile(appType: string, folderPath: string, webConfigTemplatePath: string, webConfigParameters: string) {
+    try {
+        var webConfigPath = path.join(folderPath, "web.config");
+        if (!fs.existsSync(webConfigPath)) {
+            var JSONObject = JSON.parse(webConfigParameters);
+
+            var webConfigContent = fs.readFileSync(webConfigTemplatePath, 'utf8');
+            switch (appType) {
```

Comment:
i have assumed that a template corresponding to app type will be present in WebConfigTemplates under webdeployment-common. should i change this?